src/utils.test.js

```diff
@@ -0,0 +1,51 @@
+import {
+    getPayPalWindowNamespace,
+    getBraintreeWindowNamespace,
+    hashStr,
+} from "./utils";
+
+describe("testing getPayPalWindowNamespace util function", () => {
```

Comment:
Let's follow the pattern of using the function name as the describe block value.

```suggestion
describe("getPayPalWindowNamespace", () => {
```